Source/Core/Core/IOS/USB/Bluetooth/BTEmu.cpp

```diff
@@ -474,35 +448,31 @@ bool BluetoothEmu::SendEventInquiryResponse()
               response->bdaddr[5]);
   }
 
+  inquiry_result->PayloadLength =
+      (u8)(sizeof(SHCIEventInquiryResult) - 2 +
+           (inquiry_result->num_responses * sizeof(hci_inquiry_response)));
```

Comment:
You probably want to avoid a c-style cast if you can.
```suggestion
  inquiry_result->PayloadLength =
      u8(sizeof(SHCIEventInquiryResult) - 2 +
         (inquiry_result->num_responses * sizeof(hci_inquiry_response)));
```